### PR TITLE
Soybase audit 2

### PR DESCRIPTION
**Pull request by ChenProm-USDA**
This PR mainly consist of refactoring the code base to use uikit classes instead of inline CSS to style website.  This makes everything more consistent.  There also some updates to UI/UX outlined below.  This PR also fixes some of the Pa11y errors.

**UI/UX changes consist of:**

- breaking up pages that contains a lot of data into collapsable accordion elements
- updating tables to use uikit tables
- elements that utilize the \<details\> tag have been updated to use custom uikit toggles.

**1. Use accordion to break information into smaller collapsable sections.**

- Tools/index.html
- tools/browsers/index.html
- collections/index.html
- community/soygermplasm/index.html
- help/index.html, maps/index.html,
-  projects/index.html/, resources/index.html

Example:
Screenshot 2024-11-05 at 08 43 04

**2. Update tables to use uikit tables**

- about/lgs_and_chromosomes/index.html
- about/soybase_team/index.html
- community/sbw/SBW/SBW_2013.html
- community/sbw/SBW/SBW_2014.html
- community/sbw/SBW/SBW_2016.html
- community/sbw/SBW/SBW_2017.html
- community/sgn/index.html
- community/soygermplasm/index.html
- maps/index.html
- projects/FN_mutant_details/index.html
- projects/SoyNAM/index.html
- resources/genome_info/index.html
- tools/gcvit/index.html
- tools/ontology/ontologyConversion.html
- tools/parentage/index.html
- tools/parentage/prefix_codes/index.html
- tools/snp50K/index.html
- tools/trait_list/index.html
- tools/translate/index.html

Screenshot 2024-11-05 at 10 08 04

**3. Update \<details\> to use toogles**

- community/sbw/registration.html
- tools/gcvit/index.html
- tools/ontology/index.html
- tools/search/gene.html
- tools/search/trait.html
- tools/translate/index.html

Screenshot 2024-11-05 at 10 12 04

### CODE REVIEW

File: assets/img/sbw/2025/SBW2025_Carl_Bradley.png
Comment (jd-campbell) [inline assets/img/sbw/2025/SBW2025_Carl_Bradley.png:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: assets/img/sbw/2025/SBW2025_Cory_Hirsch.png
Comment (jd-campbell) [inline assets/img/sbw/2025/SBW2025_Cory_Hirsch.png:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: assets/img/sbw/2025/SBW2025_Jan_Leach.png
Comment (jd-campbell) [inline assets/img/sbw/2025/SBW2025_Jan_Leach.png:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: assets/img/sbw/SBW_BLACK .png
Comment (jd-campbell) [inline assets/img/sbw/SBW_BLACK .png:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: assets/img/sbw/SBW_GREEN.png
Comment (jd-campbell) [inline assets/img/sbw/SBW_GREEN.png:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: assets/img/sbw/SBW_logo.png
Comment (jd-campbell) [inline assets/img/sbw/SBW_logo.png:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: assets/img/sbw/officers/SBW_Officer_Ben_Hall.png
Comment (jd-campbell) [inline assets/img/sbw/officers/SBW_Officer_Ben_Hall.png:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: assets/img/sbw/officers/SBW_Officer_Cleiton_Wartha.png
Comment (jd-campbell) [inline assets/img/sbw/officers/SBW_Officer_Cleiton_Wartha.png:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: assets/img/sbw/officers/SBW_Officer_Ian_McNish.png
Comment (jd-campbell) [inline assets/img/sbw/officers/SBW_Officer_Ian_McNish.png:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: assets/img/sbw/officers/SBW_Officer_Nicole_Choquett.jpg
Comment (jd-campbell) [inline assets/img/sbw/officers/SBW_Officer_Nicole_Choquett.jpg:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: assets/img/sbw/officers/SBW_Officer_Rex_Nelson.png
Comment (jd-campbell) [inline assets/img/sbw/officers/SBW_Officer_Rex_Nelson.png:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: assets/img/sbw/officers/SBW_officer_Aaron_Lorenz.png
Comment (jd-campbell) [inline assets/img/sbw/officers/SBW_officer_Aaron_Lorenz.png:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: assets/img/sbw/officers/SBW_officer_Keith_Rincker.png
Comment (jd-campbell) [inline assets/img/sbw/officers/SBW_officer_Keith_Rincker.png:1]: Removed from branch b/c img is too large for repo. Img is now in DS annex.

File: blog/_posts/2024-07-01-go-enrichment.md
Comment (jd-campbell) [inline blog/_posts/2024-07-01-go-enrichment.md:1]: @ChenProm-USDA On this blog half of the text is in markdown and the other half is in uikit. Please review.

File: collections/index.html
Comment (jd-campbell) [inline collections/index.html:1]: @ChenProm-USDA 
I love the accordion!

File: community/sbw/SBW/SBW_2013.html
Comment (jd-campbell) [inline community/sbw/SBW/SBW_2013.html:1]: @ChenProm-USDA 
I like the table work on this page!

File: blog/_posts/2024-07-01-go-enrichment.md
Comment (ChenProm-USDA) [reply blog/_posts/2024-07-01-go-enrichment.md:1]: on it.

Comment (ChenProm-USDA) [reply blog/_posts/2024-07-01-go-enrichment.md:1]: fixed

File: _includes/events-card.html
Comment (jd-campbell) [inline _includes/events-card.html:1]: @ChenProm-USDA 
Question: With the addition of this events-card.html -- does the way I add _posts.md to the repro for new events?

Comment (ChenProm-USDA) [reply _includes/events-card.html:1]: It shouldn't change the way you add _posts.md.

**Review by maxglycine — APPROVED** [review]
Looks good to me

**Review by jd-campbell — APPROVED** [review]
I have reviewed all 83 files changed. I think it is all good. I will merge this branch.

File: tools/snp50k/index.html
Comment (StevenCannon-USDA) [inline tools/snp50k/index.html:1]: Small error on this page: Gmo4 --> Gm04  (o should be 0)

**Review by StevenCannon-USDA — APPROVED** [review]
Very nice work, Chen. I have been through all of the pages.
See the note about a small (prior) error at tools/snp50k/index.html
And for the failed link check at `about/lgs_and_chromosomes/index.html`, where this link is failing:
  `https://pmc.ncbi.nlm.nih.gov/articles/PMC1893076/`
Try using one of these equivalent links instead:
  `https://doi.org/10.1534/genetics.107.070821`
  `https://academic.oup.com/genetics/article/176/1/685/6064539`

File: tools/snp50k/index.html
Comment (ChenProm-USDA) [reply tools/snp50k/index.html:1]: fixed

**Comment by StevenCannon-USDA** [comment]
@ChenProm-USDA - for the latest broken links, I can see nothing wrong with `https://legacy.soybase.org/gb2/gbrowse/gmax2.0/`, so I would leave that as-is. It might have been a transient server error that's resolving now.

For the Lila Vodkin staff page in the news item -- I would remove that link. The announcement is six months past, and old links will sometimes go away.

After that, I think it is time to merge this big PR! I would say -- go ahead and squash and merge!

**Pull request merged**